docs/config.md

```diff
@@ -0,0 +1,43 @@
+
+# Configuring Cerner Operations
+[[Initializing the connector]](#initializing-the-connector)  [[Additional Information]](#additional-information)
+
+## Initializing the connector
+To use the Cerner connector, add the <cerner.init> element in your configuration before carrying out any Cerner operations.
+This <cerner.init> element used to authenticates the user using OAuth2 authentication to access the Cerner registered client applications and secure sandbox.
```

Comment:
The <cerner.init> element authenticates the user using the OAuth2 authentication and allows them to access the Cerner registered client applications and secure sandbox.